Document enableNdkScopeSync option for RN (#4423)
getsentry/sentry-docs: src/platforms/react-native/troubleshooting.mdx

As this applies to **all** class and function names — not just React components — this could increase your JS bundle size.

</Note>

> ## Missing context on Android NDK events

Reviewer: ```suggestion
## Missing Context on Android NDK Events
```

> In case it is missing context on Android NDK events as breadcrumbs, user, etc..., You've to enable the [Scope Synchronization](/platforms/react-native/enriching-events/scopes/#scope-synchronization) feature.

Reviewer: ```suggestion
If the context is missing on Android NDK events such as breadcrumbs, user, and so on, you need to enable the [Scope Synchronization](/platforms/react-native/enriching-events/scopes/#scope-synchronization) feature.
```
